ios/browser/api/query_filter/BUILD.gn

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+source_set("query_filter") {
+  configs += [ "//build/config/compiler:enable_arc" ]
+
+  sources = [
+    "query_filter_utils.h",
```

Comment:
why did these get renamed to `query_filter_utils`? Or did I just not catch it before? `query_filter_` is redundant

Reply:
oic, this is the ios side, but same naming convention applies here